### PR TITLE
Abel

### PR DESCRIPTION
**Pull request by belunatic**
- Associated the group ID to create an expense form when the create expense button is pushed within the group detailed page
- disable the group dropdown when the group ID is associated with the expenses
- -changed the alert message for the amount.

### CODE REVIEW

**Review by MatthewNeie — APPROVED** [review]

File: expense-splitter/src/components/expense/CreateExpense.jsx [hunk 1]
```diff
@@ -320,6 +337,7 @@ export default function CreateExpense() {
 
           <select
             className="h-16"
+            disabled={modal.id ? true : false} //created within the group detail page
             name="group"
```
Comment (ccespedes) [inline expense-splitter/src/components/expense/CreateExpense.jsx]: Can you match the disabled group behavior found in edit expense where it utilizes cursor-not-allowed?

**Review by ccespedes — COMMENTED** [review]
Seems good. Can you see if you can match the red error text style from create expense to edit expense, create group and edit group? If not it's fine but would be nice to match it all up

File: expense-splitter/src/components/expense/CreateExpense.jsx (hunk 1 above)
Comment (singhalex) [reply expense-splitter/src/components/expense/CreateExpense.jsx]: Do we actually want to disable the drop down or just have it pre-selected?

Comment (ccespedes) [reply expense-splitter/src/components/expense/CreateExpense.jsx]: This is true, we came in hot and almost burnt the place down! 🔥
It shouldn't be disabled.

**Review by singhalex — APPROVED** [review]

**Pull request merged**